src/Driver/Driver.jl

```diff
@@ -100,14 +101,14 @@ function parse_commandline()
         "--diagnostics-interval"
             help = "interval in simulation steps for gathering diagnostics"
             arg_type = Int
-            default = 10000
+            default = 1
         "--enable-vtk"
             help = "output VTK to <output-dir> every <vtk-interval> simulation steps"
             action = :store_true
         "--vtk-interval"
             help = "interval in simulation steps for VTK output"
             arg_type = Int
-            default = 10000
+            default = 4
```

Comment:
Or this.

Since you have to explicitly `--enable-vtk` to turn this on, it isn't blowing things up in the tests now, but such a short interval is going to dump far too much data for the ordinary case (especially for testing).

Reply:
My bad